server/notification-providers/discord.js

```diff
@@ -48,7 +48,7 @@ class Discord extends NotificationProvider {
                             },
                             {
                                 name: monitorJSON["type"] === "push" ? "Service Type" : "Service URL",
-                                value: this.extractAddress(monitorJSON),
+                                value: notification.disableUrl ? "Hidden" : this.extractAddress(monitorJSON),
```

Comment:
If the url is indeed hidden, we should not have this entire section.
Seems kind of silly to hide it as is.